src/Symfony/Component/PasswordHasher/Tests/Command/UserPasswordHashCommandTest.php

```diff
@@ -67,7 +67,7 @@ public function testEncodePasswordBcrypt()
 
     public function testEncodePasswordArgon2i()
     {
-        if (!($sodium = SodiumPasswordHasher::isSupported() && !\defined('SODIUM_CRYPTO_PWHASH_ALG_ARGON2ID13')) && !\defined('PASSWORD_ARGON2I')) {
```

Comment:
I wonder if this line shouldn't be:
```php
if (!($sodium = SodiumPasswordHasher::isSupported() && \defined('SODIUM_CRYPTO_PWHASH_ALG_ARGON2ID13')) && !\defined('PASSWORD_ARGON2I'))
```
ie:
```php
if (!SodiumPasswordHasher::isSupported() || !\defined('SODIUM_CRYPTO_PWHASH_ALG_ARGON2ID13')) && !\defined('PASSWORD_ARGON2I'))
```

Reply:
I think we can just remove the whole `if` statement given we no longer have a case where argon2i might not be available (thanks to native `\PASSWORD_ARGON2I`).

Reply:
isn't it still possible to compile PHP 8 without argon2 support ? Looking into the php source code, support for argon2 is wrapped in `#if HAVE_ARGON2LIB`

Reply:
Indeed, the docs tell it is. That part of the patch can be reverted then.

> PASSWORD_ARGON2ID [...] This algorithm is only available if PHP has been compiled with Argon2 support.

Reply:
I reverted all those changed, sorry :)